From a447bb3d815c965feb4a2dbfa5edeb8680a2825e Mon Sep 17 00:00:00 2001
From: squalus <squalus@squalus.net>
Date: Mon, 29 Jan 2024 10:41:45 -0800
Subject: [PATCH 2/2] fix server path

---
 electron.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/electron.js b/electron.js
index 816ea9ecd..7a3272f21 100644
--- a/electron.js
+++ b/electron.js
@@ -7,7 +7,7 @@ const storage = require('electron-json-storage');
 const port = process.env.SERVER_PORT;
 const protocol = 'anytype';
 const remote = require('@electron/remote/main');
-const binPath = fixPathForAsarUnpack(path.join(__dirname, 'dist', `anytypeHelper${is.windows ? '.exe' : ''}`));
+const binPath = fixPathForAsarUnpack(path.join(__dirname.replace('app.asar', 'app.asar.unpacked'), 'dist', `anytypeHelper${is.windows ? '.exe' : ''}`));
 
 if (is.development) {
 	app.setPath('userData', path.join(app.getPath('userData'), '_dev'));
@@ -203,4 +203,4 @@ app.on('open-url', (e, url) => {
 		Util.send(mainWindow, 'route', Util.getRouteFromUrl(url));
 		mainWindow.show();
 	};
-});
\ No newline at end of file
+});
-- 
2.42.0

