From c2762611e6737bc8c9cac68b5ce04500873e5883 Mon Sep 17 00:00:00 2001
From: squalus <squalus@squalus.net>
Date: Mon, 29 Jan 2024 10:41:45 -0800
Subject: [PATCH 2/2] fix server path

---
 electron.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/electron.js b/electron.js
index ee925a8ce..4292f5339 100644
--- a/electron.js
+++ b/electron.js
@@ -8,7 +8,7 @@ const port = process.env.SERVER_PORT;
 const protocol = 'anytype';
 const remote = require('@electron/remote/main');
 const { installNativeMessagingHost } = require('./electron/js/lib/installNativeMessagingHost.js');
-const binPath = fixPathForAsarUnpack(path.join(__dirname, 'dist', `anytypeHelper${is.windows ? '.exe' : ''}`));
+const binPath = fixPathForAsarUnpack(path.join(__dirname.replace('app.asar', 'app.asar.unpacked'), 'dist', `anytypeHelper${is.windows ? '.exe' : ''}`));
 
 // Fix notifications app name
 if (is.windows) {
@@ -209,4 +209,4 @@ app.on('open-url', (e, url) => {
 		Util.send(mainWindow, 'route', Util.getRouteFromUrl(url));
 		mainWindow.show();
 	};
-});
\ No newline at end of file
+});
diff --git a/electron/js/lib/installNativeMessagingHost.js b/electron/js/lib/installNativeMessagingHost.js
index 22de950..8e3daba 100644
--- a/electron/js/lib/installNativeMessagingHost.js
+++ b/electron/js/lib/installNativeMessagingHost.js
@@ -23,7 +23,7 @@ const EXE_PATH = app.getPath('exe');

 const getManifestPath = () => {
	const fn = `nativeMessagingHost${is.windows ? '.exe' : ''}`;
-	return path.join(fixPathForAsarUnpack(__dirname), '..', '..', '..', 'dist', fn);
+	return path.join(fixPathForAsarUnpack(__dirname), '..', '..', '..', 'dist', fn).replace('app.asar', 'app.asar.unpacked');
 };

 const getHomeDir = () => {
diff --git a/electron/js/window.js b/electron/js/window.js
index 1f59ffe..893a8dc 100644
--- a/electron/js/window.js
+++ b/electron/js/window.js
@@ -29,7 +29,7 @@ class WindowManager {
			show: false,
			titleBarStyle: 'hidden-inset',
			webPreferences: {
-				preload: fixPathForAsarUnpack(path.join(Util.electronPath(), 'js', 'preload.js')),
+				preload: fixPathForAsarUnpack(path.join(Util.electronPath(), 'js', 'preload.js')).replace('app.asar', 'app.asar.unpacked'),
			},
		}, param);


-- 
2.42.0

