From 11d027b6330bbebcb1e94640c0266a3b6e4b2142 Mon Sep 17 00:00:00 2001
From: squalus <squalus@squalus.net>
Date: Mon, 29 Jan 2024 10:41:45 -0800
Subject: [PATCH 2/2] fix server path

---
 electron.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/electron.js b/electron.js
index c5d6760882..dd3c28c412 100644
--- a/electron.js
+++ b/electron.js
@@ -8,7 +8,7 @@ const port = process.env.SERVER_PORT;
 const protocol = 'anytype';
 const remote = require('@electron/remote/main');
 const { installNativeMessagingHost } = require('./electron/js/lib/installNativeMessagingHost.js');
-const binPath = fixPathForAsarUnpack(path.join(__dirname, 'dist', `anytypeHelper${is.windows ? '.exe' : ''}`));
+const binPath = fixPathForAsarUnpack(path.join(__dirname.replace('app.asar', 'app.asar.unpacked'), 'dist', `anytypeHelper${is.windows ? '.exe' : ''}`));
 const Store = require('electron-store');
 const suffix = app.isPackaged ? '' : 'dev';
 const store = new Store({ name: [ 'localStorage', suffix ].join('-') });
-- 
2.49.0

