From e9317b8f787e80740737b374476a71718c0d12fc Mon Sep 17 00:00:00 2001
From: squalus <squalus@squalus.net>
Date: Mon, 29 Jan 2024 10:41:45 -0800
Subject: [PATCH 2/2] fix server path

---
 electron.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/electron.js b/electron.js
index d06dd0481..d1ccc5c9d 100644
--- a/electron.js
+++ b/electron.js
@@ -8,7 +8,7 @@ const port = process.env.SERVER_PORT;
 const protocol = 'anytype';
 const remote = require('@electron/remote/main');
 const { installNativeMessagingHost } = require('./electron/js/lib/installNativeMessagingHost.js');
-const binPath = fixPathForAsarUnpack(path.join(__dirname, 'dist', `anytypeHelper${is.windows ? '.exe' : ''}`));
+const binPath = fixPathForAsarUnpack(path.join(__dirname.replace('app.asar', 'app.asar.unpacked'), 'dist', `anytypeHelper${is.windows ? '.exe' : ''}`));
 
 // Fix notifications app name
 if (is.windows) {
@@ -221,4 +221,4 @@ app.on('open-url', (e, url) => {
 		Util.send(mainWindow, 'route', Util.getRouteFromUrl(url));
 		mainWindow.show();
 	};
-});
\ No newline at end of file
+});
-- 
2.44.1

