From 620009374759144f85c60d80a6e2c6e76d52f676 Mon Sep 17 00:00:00 2001
From: squalus <squalus@squalus.net>
Date: Mon, 29 Jan 2024 10:41:45 -0800
Subject: [PATCH 2/2] fix server path

---
 electron.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/electron.js b/electron.js
index 9f58f11d70..fc9b0396a1 100644
--- a/electron.js
+++ b/electron.js
@@ -7,7 +7,7 @@ const storage = require('electron-json-storage');
 const protocol = 'anytype';
 const remote = require('@electron/remote/main');
 const { installNativeMessagingHost } = require('./electron/js/lib/installNativeMessagingHost.js');
-const binPath = fixPathForAsarUnpack(path.join(__dirname, 'dist', `anytypeHelper${is.windows ? '.exe' : ''}`));
+const binPath = fixPathForAsarUnpack(path.join(__dirname.replace('app.asar', 'app.asar.unpacked'), 'dist', `anytypeHelper${is.windows ? '.exe' : ''}`));
 const Store = require('electron-store');
 const suffix = app.isPackaged ? '' : 'dev';
 const store = new Store({ name: [ 'localStorage', suffix ].join('-') });
-- 
2.50.1

